src/test/java/com/wikia/webdriver/elements/fandom/components/LoadMore.java

```diff
@@ -0,0 +1,35 @@
+package com.wikia.webdriver.elements.fandom.components;
+
+import com.gargoylesoftware.htmlunit.ElementNotFoundException;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+
+/**
+ * Created by liz_lux on 1/26/17.
+ */
+public class LoadMore extends BasePageObject {
+
+    @FindBy(css = ".more-stories .load-more-btn")
+    private WebElement loadMoreButton;
+
+    public boolean isButtonDisplayed() {
+        try {
+            return loadMoreButton.isDisplayed();
+        } catch (ElementNotFoundException e) {
+            return false;
+        }
+    }
+
+    public boolean areMorePostsDisplayed() {
+        try {
+            loadMoreButton.click();
+            WebElement element = wait.forElementPresent(By.cssSelector("[tracking-label='more-top-stories.item-15']"));
+            return element.isDisplayed();
+        } catch (ElementNotFoundException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)